xmtp_mls/src/groups/scoped_client.rs

```diff
@@ -384,21 +384,21 @@ where
             .await
     }
 
-    async fn get_association_state<'a>(
+    async fn get_association_state(
         &self,
         conn: &DbConnection,
-        inbox_id: InboxIdRef<'a>,
+        inbox_id: InboxIdRef<'_>,
```

Comment:
```suggestion
        inbox_id: InboxIdRef,
```
Would this work?

Reply:
without the anonymous lifetime it gives a compiler error:

```
implicit elided lifetime not allowed here
expected lifetime parameterrustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B0%5D?0#file%3A%2F%2F%2FUsers%2Fcameronvoell%2FXMTP%2Flibxmtp%2Fxmtp_mls%2Fsrc%2Fgroups%2Fscoped_client.rs)
scoped_client.rs(390, 29): indicate the anonymous lifetime: `<'_>`
```